src/myjarvis/domain/entities/ai_agent.py

```
@@ -1,17 +1,146 @@
+
+from src.myjarvis.domain.value_objects import (
+    AgentId,
+    NodeId,
+    UserId,
+)
+from src.myjarvis.domain.value_objects.ai_agent_name import AgentName
+from src.myjarvis.domain.value_objects.llm_model import LlmModel
+
+
+@dataclass
+class AIAgent:
```

Pylint (3.3.7)

[refactor] Too many instance attributes (11/7)

(R0902)